accounting/accounting_cli_functions.py

```diff
@@ -248,6 +248,196 @@ def view_jobs_before_end_time(conn, time_before, output_file):
     return job_records
 
 
+def add_bank(conn, bank, shares, parent_bank=""):
+    parent_bank_shares = 0
+    total_sub_bank_shares = 0
+
+    # if the parent bank is not "", that means the account
+    # trying to be added wants to be placed under a parent bank
+    if parent_bank != "":
+        try:
+            select_stmt = "SELECT shares FROM bank_table where bank=?"
+            dataframe = pd.read_sql_query(select_stmt, conn, params=(parent_bank,))
+            # if length of dataframe is 0, that means the parent bank wasn't found
+            if len(dataframe.index) == 0:
+                print("Parent account not found in bank table")
+                sys.exit(-1)
```

Comment:
Typically, you want fatal errors like this to get printed to stderr, and you don't want imported functions to fatally kill the interpreter with `sys.exit`.  My recommendation would be to raise an appropriately named exception (e.g., `RuntimeException`) with the string as the message within the exception.  Then have the top-level caller - `main` in this case - catch the exception, print the message to stderr and exit the interpreter with an exit code.

Same message applies elsewhere in this file that print and sys.exit are paired for an error.